lib/task-data/schemas/clear-sel-by-usage.json

```diff
@@ -0,0 +1,23 @@
+{
+    "copyright": "Copyright 2017 Dell Inc. or its subsidiaries. All Rights Reserved.",
+    "definitions": {
+        "selForceClearFlag": {
+            "type": "boolean",
+            "description": "User specifies whether force to clear sel when sel entry is almost full"
+        },
+        "maxSelUsagePercent": {
```

Comment:
Also this is a typo, the code use variable `maxSelUsage` rather than `maxSelUsagePercent`